tempodb/encoding/versioned.go

```diff
@@ -85,7 +85,7 @@ func LatestEncoding() VersionedEncoding {
 func AllEncodings() []VersionedEncoding {
 	return []VersionedEncoding{
 		v2.Encoding{},
-		vparquet.Encoding{},
+		//	vparquet.Encoding{},
```

Comment:
this will help with tests. good call.

Reply:
can we remove it instead of commenting it out?